pkg/manifests/types.go

```diff
@@ -601,6 +601,9 @@ type AlertmanagerUserWorkloadConfig struct {
 // The `PrometheusRestrictedConfig` resource defines the settings for the
 // Prometheus component that monitors user-defined projects.
 type PrometheusRestrictedConfig struct {
+	// Configures Interval between consecutive scrapes.
```

Comment:
```suggestion
	// Configures the default interval between consecutive scrapes in case the ServiceMonitor or PodMonitor resource doesn't specify any value.
	// Allowed values are between 10s and 300s (5m).
```

Reply:
I would just add to this suggestion to replace `doesn't` for `does not`. Thanks!